src/app/dataset-view/dataset.component.scss

```diff
@@ -1,145 +1,149 @@
 @import "var";
 
-.mat-button-toggle-group {
-    width: fit-content;
-
-    .mat-button-toggle-standalone.mat-button-toggle-appearance-standard,
-    .mat-button-toggle-group-appearance-standard {
-        border-radius: 0 !important;
-    }
-}
-
-#dataset-view-history {
-    overflow: visible;
-}
-
-.datatab-expanding-row {
-    background: #fff;
-    border-top: 1px solid rgb(0 0 0 / 12%);
-    box-shadow: 0 3px 3px rgb(0 0 0 / 12%);
-    transition: margin 0.2s cubic-bezier(0.4, 0, 0.2, 1);
-    will-change: margin;
-
-    &-header {
-        color: #000;
-        font-size: 13px;
-        font-weight: 400;
-        line-height: 20px;
-        cursor: pointer;
-        display: flex;
-        padding: 4px 10px;
-
-        &-content {
-            flex-grow: 1;
-            max-width: calc(100% - 24px);
+:host {
+    ::ng-deep {
+        .mat-button-toggle-group {
+            width: fit-content;
+
+            .mat-button-toggle-standalone.mat-button-toggle-appearance-standard,
+            .mat-button-toggle-group-appearance-standard {
+                border-radius: 0 !important;
+            }
         }
 
-        .datatab-activity-row-summary {
-            display: flex;
+        #dataset-view-history {
+            overflow: visible;
         }
```

Comment:
These styles should be in the history.component

Reply:
Moved to history.component